src/plugins/desktop/README.md

```diff
@@ -24,13 +24,13 @@ The package is called `libelektra5-extra`.
 To mount the plugin please use:
 
 ```sh
-sudo kdb mount --resolver noresolver none system:/info/desktop desktop
```

Comment:
sudo should be always added to `kdb mount`.

Reply:
Thank you for mentioning this. The problem I encountered is that `sudo` was not available in the test environment. Does the test environment of the CI support `sudo`?

Reply:
The shell recorder should strip `sudo `:

https://github.com/ElektraInitiative/libelektra/blob/43499ed730d57bac812b0a04efbf355a35371b48/tests/shell/shell_recorder/tutorial_wrapper/markdown_shell_recorder.sh#L29

Was it maybe that there was a tab after the sudo or similar?

Reply:
This makes sense, thank you very much for clarifying. I will look into that.